### PR TITLE
trigger fix

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: cli/commands.go [hunk 1]
```diff
@@ -65,6 +65,8 @@ var (
 				flPath,
 				flConfig,
 				flConfigBase64,
+				flInventory,
```
Comment (godrei) [inline cli/commands.go]: should not be required

File: cli/run_util_test.go [hunk 2]
```diff
@@ -10,6 +10,177 @@ import (
...
+		require.NoError(t, err)
+		require.Equal(t, true, pr)
+	}
+
+	t.Log("prGlobalFlag: false, prModeEnv: 'false', prIDEnv: '', secrets: true => true")
+	{
+		inventoryStr := `
+envs:
+- PR: "true"
+- PULL_REQUEST_ID: "-"
```
Comment (godrei) [inline cli/run_util_test.go]: make it empty here

File: cli/run_util_test.go [hunk 3]
```diff
@@ -10,6 +10,177 @@ import (
...
+		ci, err := isCIMode(true, "", inventory.Envs)
+		require.NoError(t, err)
+		require.Equal(t, true, ci)
+	}
+
+	t.Log("ciGlobalFlag: false, ciModeEnv: 'true' secrets: false => true")
+	{
+		inventoryStr := `
+envs:
+- CI: "-"
```
Comment (godrei) [inline cli/run_util_test.go]: empty too

File: bitrise-integration.yml [hunk 4]
```diff
@@ -15,6 +15,16 @@ trigger_map:
   is_pull_request_allowed: true
   workflow: exit_code_test_sippable_ok
 
+- pattern: "on_pr"
```
Comment (viktorbenei) [inline bitrise-integration.yml]: `pr_allowed`

File: bitrise-integration-secrets.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+envs:
```
Comment (viktorbenei) [inline bitrise-integration-secrets.yml]: postfix this file with `-pr.yml`

File: bitrise-integration.yml [hunk 6]
```diff
@@ -15,6 +15,16 @@ trigger_map:
...
+  is_pull_request_allowed: true
+  workflow: on_pr
+- pattern: "on_push"
```
Comment (viktorbenei) [inline bitrise-integration.yml]: `only_code_push`

File: bitrise.yml [hunk 7]
```diff
@@ -194,6 +194,75 @@ workflows:
...
               exit 1
             fi
+    - script:
+        title: Trigger check
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -e
+
+            export PR=""
```
Comment (viktorbenei) [inline bitrise.yml]: add comment : `# PR mode : from secrets`

File: cli/run_util_test.go [hunk 8]
```diff
@@ -2,14 +2,244 @@ package cli
...
+	"github.com/bitrise-io/bitrise/configs"
 	envmanModels "github.com/bitrise-io/envman/models"
 	"github.com/stretchr/testify/require"
 )
 
+func TestIsPRMode(t *testing.T) {
+	prModeEnv := os.Getenv(configs.PRModeEnvKey)
+	prIDEnv := os.Getenv(configs.PullRequestIDEnvKey)
+
+	defer func() {
```
Comment (viktorbenei) [inline cli/run_util_test.go]: add comment: `// cleanup Envs after these tests`

File: cli/run_util_test.go [hunk 9]
```diff
@@ -2,14 +2,244 @@ package cli
...
+		require.NoError(t, os.Setenv(configs.PRModeEnvKey, prModeEnv))
+		require.NoError(t, os.Setenv(configs.PullRequestIDEnvKey, prIDEnv))
+	}()
+
+	t.Log("Should be: false - prGlobalFlag: false, prModeEnv: '', prIDEnv: ''")
```
Comment (viktorbenei) [inline cli/run_util_test.go]: format: `Should be false for: ...`

Comment (viktorbenei) [inline cli/run_util_test.go]: or: `Should be false for => `

**Pull request merged**